### PR TITLE
Implement i18n and l10n in russian

### PR DESCRIPTION
**Pull request by AlenaSetukha**
You can switch localization manualy in main.ReaBoo class in lines 102, 103. Use `pybabel compile -D reaboo -i po/ru_RU.UTF-8/LC_MESSAGES/reaboo.po -l ru_RU.UTF-8 -d po` command to creat a `.mo` file.

### CODE REVIEW

File: main.py
Comment (ruslkhay) [inline main.py:1]: Have you checked flake8 and pydocstyle on this one?

File: babel.cfg
Comment (ruslkhay) [inline babel.cfg:1]: What is this file?

**Review by ruslkhay — COMMENTED** [review]
Nice work. If flake8 and pydocstyle are ok, then I'll merge.

File: babel.cfg
Comment (AlenaSetukha) [reply babel.cfg:1]: This is a configuration file for babel. From the box `pybabel init`command checks only `.py` files for `gettext` translation function. But all of text is in `reaboo.kv` file, so I'd to extend basic configuration.

File: main.py
Comment (AlenaSetukha) [reply main.py:1]: Yes, but now I've missed one trailing whitespace in line 101.

File: reaboo.kv
Comment (AlenaSetukha) [inline reaboo.kv:1]: You've erased to add app._() function onto "New word" and "Type a new word".

**Review by ruslkhay — APPROVED** [review]
Congrats! Only auto-deployment left.

**Pull request merged**